core/src/main/scala/cats/syntax/applicativeError.scala

```diff
@@ -34,4 +34,7 @@ final class ApplicativeErrorOps[F[_], E, A](val fa: F[A]) extends AnyVal {
 
   def recoverWith(pf: PartialFunction[E, F[A]])(implicit F: ApplicativeError[F, E]): F[A] =
     F.recoverWith(fa)(pf)
+
+  def onError(fa: F[A])(pf: PartialFunction[E, F[Unit]])(implicit F: ApplicativeError[F, E]): F[A] =
+    F.onError(fa)(pf)
```

Comment:
I can't see any doctests for the other methods on ApplicativeError, apart from a scaladoc example for `fromEither`.  Would you happen to have an example at hand of how syntax forwarders are tested in cats via doctests  ?